pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -64,6 +64,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to cloud providers.
+	err = c.Watch(
+		&ProviderSource{
```

Comment:
@jortel when this updates every 30 seconds, does it always update the MigStorage? Trying to figure out if this will be triggered reconciles all the time still.

Reply:
I figured this out. It only enqueues a reconcile when there's an issue, meaning this shouldn't cause problems.